lib/bolt/logger.rb

```diff
@@ -67,6 +68,10 @@ def self.configure(destinations, color)
       end
     end
 
+    def self.analytics(analytics = Bolt::Analytics::NoopClient.new)
```

Comment:
I'm not clear what this method is supposed to do. Is it a getter or a setter or both?

Reply:
It's meant to just be a setter, but calling it `set_analytics` makes rubocop unhappy. There's probably a better way.

Reply:
In that case, it should be `def self.analytics=(analytics)`. No default value for the argument then.